apps/meteor/client/views/admin/apps/AppSetting.tsx

```diff
@@ -66,7 +89,7 @@ function AppSetting({ appSetting, onChange, value, ...props }) {
 			value={value}
 			onChangeValue={onChange}
 			_id={id}
-			values={translatedValues}
+			{...({ values: translatedValues } as any)}
```

Comment:
> any

![image](https://user-images.githubusercontent.com/2263066/185701484-37ebca8d-eb86-4e93-baf0-0b814b33f2a8.png)

Reply:
That was acctually intentional... but fixed it anyway.